pkg/accessibility/accessibility.go

```diff
@@ -0,0 +1,29 @@
+package accessibility
+
+import (
+	"os"
+
+	"github.com/cli/go-gh/v2/pkg/config"
+)
+
+const ACCESSIBILITY_ENV = "ACCESSIBILITY"
```

Comment:
Are there any concerns aligning with the `ACCESSIBLE` env var used by [`charmbracelet/huh`](https://github.com/charmbracelet/huh#accessibility) for accessible prompting?

```suggestion
const ACCESSIBILITY_ENV = "ACCESSIBLE"
```

Reply:
Ah, good call. We should probably suffix this with `GH_` as well 👍